test/api/connectors/ConnectorSpec.scala

```diff
@@ -115,6 +116,10 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
     "Gov-Test-Scenario" -> "DEFAULT"
   )
 
+  val desIfMigrationEnabledConfig: Configuration = Configuration("desIf_Migration.enabled" -> true)
+
+  val desIfMigrationDisabledConfig: Configuration = Configuration("desIf_Migration.enabled" -> false)
+
```

Comment:
Most connectors don't need the feature switch to control the migration (they're already using IFS). So I think it is clearer if the tests themselves set this configuration i.e. that we don't change this class. (See corresponding comments against the delete connector test itself.) 

(A method _could_ be added in here for the tests to explicitly call to set the switch but since there is only one applicable connector in this API it's not really worth it.)